src/features/delegation/hooks/useDelegators.ts

```diff
@@ -3,27 +3,32 @@ import { useQuery } from '@tanstack/react-query';
 import { useToastError } from 'src/components/notifications/useToastError';
 import { GCTime, StaleTime } from 'src/config/consts';
 import { Addresses } from 'src/config/contracts';
+import { Delegatee } from 'src/features/delegation/types';
 import { queryCeloBlockscoutLogs } from 'src/features/explorers/blockscout';
 import { TransactionLog } from 'src/features/explorers/types';
+import { fetchStCELOStakers } from 'src/features/staking/stCELO/hooks/useStCELO';
 import { logger } from 'src/utils/logger';
 import { Address, PublicClient, decodeEventLog, encodeEventTopics } from 'viem';
 import { usePublicClient } from 'wagmi';
 
 /**
  * Fetches all delegators for a given delegate address
  */
-export function useDelegators(delegateAddress?: Address) {
+export function useDelegators(delegatee?: Delegatee) {
```

Comment:
since the rules of hooks force if logic to be inside the hook lets rename to useDelegatorOrStakers